pkgs/top-level/python-packages.nix

```diff
@@ -9203,6 +9203,25 @@ let
     };
   };
 
+  mpv = buildPythonPackage rec {
+    name = "mpv-0.1";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/m/mpv/${name}.tar.gz";
+      md5 = "48114a00be2dc8898a88b3a253362fae";
+    };
```

Comment:
Both PyPI and nixpkgs repository still use md5 a lot. That is not good. Why?
The idea of the hash is to know for sure you have the file that you want to have. If a source file in the cache or upstream was changed, you want to know that.

Unfortunately, multiple files can be created with the same hash. Doing this on purpose is a collision attack. A collision attack with an md5 hash can be done in mere seconds if I am correct. With SHA-1 and SHA-2(56) it will take [a lot longer](https://www.schneier.com/blog/archives/2012/10/when_will_we_se.html). 

Last month [support for SHA-512](https://github.com/NixOS/nix/commit/6f1743b1a5116ca57a60b481ee4083c891b7a334) was added to Nix. I suppose we soon should start using SHA-512 then.